world-bank/src/Code.js

```diff
@@ -0,0 +1,111 @@
+var cc = DataStudioApp.createCommunityConnector();
+
+function getAuthType() {
+  return cc
+    .newAuthTypeResponse()
+    .setAuthType(cc.AuthType.NONE)
+    .build();
+}
+
+function getConfig(request) {
+  return {
+    configParams: [],
```

Comment:
Since this is an example/showcase connector, we should add an INFO element asking the user to click Connect.